src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/StartupProjectRegistrar.cs

```diff
@@ -69,12 +73,8 @@ public Task InitializeAsync()
 
         protected override async Task InitializeCoreAsync(CancellationToken cancellationToken)
         {
-            IProjectGuidService2 projectGuidService = ProjectGuidServices.FirstOrDefault()?.Value as IProjectGuidService2;
```

Comment:
Should `ProjectGuidServices` be removed?

Reply:
Yes it should.

Reply:
Fixed.